examples/compiled/area_cumulative_freq.vg.json

```diff
@@ -12,13 +12,10 @@
       "format": {"type": "json"},
       "transform": [
         {
-          "type": "window",
-          "params": [null],
+          "type": "joinaggregate",
```

Comment:
Cumulative frequency should not use join aggregate.  

Window with frame = `[null, 0]` makes more sense. (See SVG)

Reply:
Ups, my mistake.

Reply:
The code should only do it when the frame is `[null, null]`

Reply:
This was a bug in the code. Fixed.